tensorboard/plugins/mesh/summary.py

```diff
@@ -79,6 +84,16 @@ def _get_json_config(config_dict):
   return json_config
 
 
+def _get_components_bitmap(tensors):
+  """Creates bitmap for all existing components of the summary."""
```

Comment:
Please document the argument type, especially because the name is kind
of misleading: it’s called `tensors`, but it’s actually a list of pairs
that includes tensors, and you don’t actually care about the tensors.